bundles/net.tourbook/src/de/byteholder/geoclipse/mapprovider/DialogMPCustom.java

```diff
@@ -808,19 +776,34 @@ public void widgetSelected(final SelectionEvent e) {
                   onSelectZoomSpinnerMax();
                }
             });
-            _spinMaxZoom.addModifyListener(new ModifyListener() {
-               @Override
-               public void modifyText(final ModifyEvent e) {
-                  if (_isInitUI) {
-                     return;
-                  }
-                  onSelectZoomSpinnerMax();
+            _spinMaxZoom.addModifyListener(event -> {
+               if (_isInitUI) {
+                  return;
                }
+               onSelectZoomSpinnerMax();
             });
          }
       }
    }
 
+   private void createUI230UserAgent(final Composite parent) {
+
+      final Composite container = new Composite(parent, SWT.NONE);
+      GridDataFactory.fillDefaults().applyTo(container);
+      GridLayoutFactory.fillDefaults().numColumns(2).applyTo(container);

```

Comment:
Fixed
![image](https://user-images.githubusercontent.com/25235665/105534809-48793c80-5cab-11eb-929c-53c13adaf953.png)

Reply:
>The image/png label should also a little be more located to the left but have not yet investigated why it isn't. This is old code which I wrote about 10 years ago, and, don't touch a running system when not needed.

I think a bigger issue is that when you open the UI, the map is too much in the right
See in 20.12.0:
![image](https://user-images.githubusercontent.com/25235665/105535114-ab6ad380-5cab-11eb-8ece-a67e40f6b3a8.png)

Reply:
When starting from scratch, then the dialog looks like this and the map size can be adjusted

![mt-custom-map-provider-dialog](https://user-images.githubusercontent.com/1283445/105570864-094ef800-5d4c-11eb-9155-f20d4cd20b00.png)

Reply:
Yes, it can be adjusted but it's not very user-friendly for the first impression.
The same issue happens when first opening the marker modification dialog

![image](https://user-images.githubusercontent.com/25235665/105609691-33aeae80-5d68-11eb-9275-6fe0af2935fd.png)

Reply:
I could verify this issues (when starting from scratch) for the marker editor dialog and will fix it.

I remember, that there was this issue, somewhere but this is now years away. Because I do test MT mostly with my production workspace this did not appear again.

Reply:
I actually test MT mainly with fresh database and configuration but also with my production database.